packages/compose/README.md

```diff
@@ -141,6 +141,8 @@ _Related_
 
 ### pure
 
+> **Deprecated** Use `memo` or `PureComponent` instead.
```

Comment:
Also, we don't expose `PureComponent` at all in `@wordpress/element`, do we? So it can be confusing to mention it in the first place.

Reply:
The only reason the `PureComponent` was omitted was because of this HoC. See https://github.com/WordPress/gutenberg/issues/33674#issuecomment-1862790710.